packages/server/graphql/mutations/upgradeToTeamTier.ts

```diff
@@ -53,7 +62,12 @@ export default {
     const viewer = await dataLoader.get('users').load(viewerId)
     const {email} = viewer!
     try {
-      await upgradeToTeamTier(orgId, stripeToken, email, dataLoader)
+      // TODO: remove upgradeToTeamTierOld once we rollout the new checkout flow: https://github.com/ParabolInc/parabol/milestone/150
+      if (paymentMethodId) {
+        await upgradeToTeamTier(orgId, paymentMethodId, email, dataLoader)
```

Comment:
`upgradeToTeamTier` changed quite a bit now that we're using Stripe Elements and use the paymentMethodId rather than creating a stripe token using the credit card details the user gives us. 

I felt that the simplest move was to rename `upgradeToTeamTier` to `upgradeToTeamTierOld` and create a new `upgradeToTeamTier` helper function. We can remove the old one when we rollout the new checkout flow.